Tensile/KernelWriter.py

```diff
@@ -320,13 +321,66 @@ def makeSubIterSchedule(self, kernel, localReadCode, globalReadCode, \
       # tack on the pointer and mac code:
       iterCode.addCode(pointerCode)
       iterCode.addCode(waitCode)
+      iterCode.addCode(packCode)
       iterCode.addCode(macIterCode)
     elif self.scheduleIterAlg == 2:
-      # fuss goes here:
+    # SIA2 use only 1 iteration and separate compute and fetch by raising compute priority
+    # 2 workgroup interleave, while WG0/WG1 doing compute, WG1/WG0 doing fetch
+    # EPS need to be 1, or valu instruction will break interleave
+      iterCode.addCode(globalReadCode)
+      iterCode.addCode(localReadCode)
+      iterCode.addCode(waitCode)
+
+      # interleave pack code
+      packItem = packCode.flatitems()
+      packCount = 0
+      packInstPerIter = (1 + self.numRowInsts * self.numColInsts)*(kernel["MatrixInstK"]//2)
+      while packItem:
+        if packCount < packInstPerIter:
+          item = packItem.pop(0)
+          iterCode.addCode(item)
+          packCount = packCount + 1
+        else:
+          break
+      
+      if packCount == 0:
+        tmpVgpr = self.vgprPool.checkOut(1)
+        iterCode.addInst("v_mov_b32 ","v%u"%(tmpVgpr),"0x0","valu operation to have different priority")
+        self.vgprPool.checkIn(tmpVgpr)
+
+      iterCode.addInst("s_setprio ","3","Raise priority while processing macs")
+      macIterItem = macIterCode.flatitems()
+      # pop the first code which is s_nop 1 for packing
+      if kernel["MatrixInstruction"] and \
+      (kernel["ProblemType"]["DataType"].isBFloat16() or kernel["ProblemType"]["DataType"].isHalf()):
+        item = macIterItem.pop(0)
+        iterCode.addCode(item)
+
+      while macIterItem:
+        item = macIterItem.pop(0)
+        iterCode.addCode(item)
+        packCount = 0
+        while packItem:
+          if packCount < 8:
+            item = packItem.pop(0)
+            iterCode.addCode(item)
+            packCount = packCount + 1
+          else:
+            break
+        if packCount > 0:
+          iterCode.addInst("s_nop ","1","VALU packing writes to be consumed by matrix instruction")  
+
+      iterCode.addInst("s_setprio ","1","Raise priority while processing macs")
+      iterCode.addCode(localWriteCode)
+      iterCode.addCode(pointerCode)
+      iterCode.addInst("s_setprio ","2","Raise priority while processing macs")
```

Comment:
not clear why we raising and lowering wave's priority here.   

s_setprio 3
s_setprio 1
s_setprio 2 

Are we raising priority for PACK mac code and lowering for non SIMD and raising again. ?

what other 'waves in same SIMD' are doing while PACK code is running?